tests/test_dagmc_query.py

```diff
@@ -1,68 +1,186 @@
 from pymoab import core, types
 from pymoab.rng import Range
-import dagmc_stats.DagmcFile as ds
+import dagmc_stats.DagmcFile as df
 import dagmc_stats.DagmcQuery as dq
 import pandas as pd
 import numpy as np
 import warnings
 
-test_env = [{'input_file': 'tests/3vols.h5m'}, {
-    'input_file': 'tests/single-cube.h5m'}, {'input_file': 'tests/pyramid.h5m'}]
+test_env = {'three_vols': 'tests/3vols.h5m',
+            'single_cube': 'tests/single-cube.h5m', 'pyramid': 'tests/pyramid.h5m'}
 
 
-def test_get_tris_vol():
+def test_pandas_data_frame():
+    """Tests the initialization of pandas data frames
+    """
+    single_cube = df.DagmcFile(test_env['single_cube'])
+    single_cube_query = dq.DagmcQuery(single_cube)
+    exp_vert_data = pd.DataFrame()
+    assert(single_cube_query._vert_data.equals(exp_vert_data))
+
+    exp_tri_data = pd.DataFrame()
+    assert(single_cube_query._tri_data.equals(exp_tri_data))
+
+    exp_surf_data = pd.DataFrame()
+    assert(single_cube_query._surf_data.equals(exp_surf_data))
+
+    exp_vol_data = pd.DataFrame()
+    assert(single_cube_query._vol_data.equals(exp_vol_data))
+
+def test_get_entities_rootset():
+    """Tests the get_entities function for rootset
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    three_vols_query = dq.DagmcQuery(three_vols)
+    exp = [three_vols.root_set]
+    assert(three_vols_query.meshset_lst == exp)
+
+def test_get_entities_vol():
+    """Tests the get_entities function for volume meshset
     """
-    Tests the get_tris function for volume meshset
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    vol = three_vols._my_moab_core.get_entities_by_type_and_tag(
+        three_vols.root_set, types.MBENTITYSET, three_vols.dagmc_tags['geom_dim'], [3])[0]
```

Comment:
readability/maintainability: let's use the entityset ranges we've already extracted.  This change can be used in a few other tests below- please review
```suggestion
    vol = three_vols.entityset_ranges[3][0]
```